styx-common/src/main/java/com/spotify/styx/model/data/Execution.java

```diff
@@ -41,19 +42,44 @@
   @JsonProperty
   public abstract Optional<String> commitSha();
 
+  @JsonProperty
+  public abstract Optional<FlyteExecConf> flyteExecConf();
```

Comment:
except for that do we need to keep a reference of the flyte execution anywhere?

Reply:
@ckiosidis Can you elaborate more? I didn't understand your comment

Reply:
I wonder if we have all the information needed in the flyteExecConf in order to poll for the status of the wf in flyte. I can look into it after we merge this

Reply:
OK so I will resolve this as flyte-exec-id is not needed for this particular PR, If we need to add or change code later so be it